### ISSUE TITLE
Feedback requested!!

### ISSUE THREAD

**Issue by polterguy** [polterguy/phosphorusfive#17]
Whoa, do I have something cool to show you guys!! [Read this](https://gaiasoul.com/2017/08/17/phosphorus-five-zeus-developers-preview-release/), and check out the [latest goods](https://github.com/polterguy/phosphorusfive/releases).

For the record, it's a BETA release, but I am 100% certain about, that this will kick some ass, when it's done. I am working on integrated Windows, Mac OS, and Linux installers - Which implies we'll potentially easily be able to reach millions ...!!

Basically, there's an Open Source "AppStore" integrated into the default core download, that allows you to cryptographically secured, distribute your apps as you see fit! It also has PayPal integration, allowing you to even sell proprietary apps through your Bazar if you wish!

While the system is still a simple framework, easily consumed as such, by simply deleting the "bazar" and "desktop" folders, and creating your own "startup.hl" file from scratch, if you wish.

**Comment by polterguy** [comment]
Check out [this one](https://www.youtube.com/watch?v=_nFebKPjA3Y).
